packages/react-button/package.json

```diff
@@ -47,9 +47,7 @@
   },
   "dependencies": {
     "@fluentui/keyboard-key": "^0.2.14",
-    "@fluentui/react-icons-mdl2": "^1.0.1",
     "@fluentui/react-make-styles": "^9.0.0-alpha.4",
-    "@fluentui/react-shared-contexts": "^1.0.0-beta.10",
```

Comment:
These deps were simply not used at all 😮

Reply:
FYI, this happened because a bunch of things were deleted in #17060. Once some of them come back we might want to find out how to deal with this.

Reply:
If we're removing this, should react-shared-contexts be released as 1.0 now? Or entirely removed for the time being (and remove usage from v8 as well)?